.circleci/config.yml

```diff
@@ -241,6 +245,12 @@ references:
       command: |
         envdir /secrets aws s3 cp --acl public-read ${PACKAGE_TARBALL:?} s3://${S3_RELEASES_BUCKET:?}/
 
+  upload_latest_artifacts: &upload_latest_artifacts
```

Comment:
Please create a proper CircleCI command, there is no full config migration scheduled, so we have to do it gradually when adding/changing it

Reply:
Then it might be better to refactor the other command too, as we could use the track of gradual changes